apps/passport-server/migrations/75_podbox_manual_tickets.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE if not exists pipeline_manual_tickets (
+  pipelineId UUID not null,
+  manualTicket JSONB not null
```

Comment:
I don't think we *need* to care about this, but we use snake_case for all of our other column names.

Reply:
fixed